### ISSUE TITLE
customizing the interval for horizontal values

### ISSUE THREAD

**Issue by Akshata001** [jjoe64/GraphView#622]
I am loading the data on the graph dynamically. There is Time (X axis)-> Value (Y axis)  line graph. I am getting the time which is to show on X axis. i.e. 12:01:55, 12:02:55, 12:03:55 and so on.. Now i want to plot the time interval of 1 minute, which should be 12:01:00, 12:02:00, 12:03:00 and so on. But library plotting the value as 12:01:05, 12:01:10, 12:01:15, 12:01:20 and so on , Which is having only 5 sec of interval. 

Please help me out, how can I customize those intervals on horizontal axis.

**Comment by Faiizii** [comment]
yes how do we customize that ?

**Comment by Knoxie** [comment]
Yeah, I'd like to be able to set the exact interval of the X-axis
